docs/news.d/config_store.rst

```diff
@@ -0,0 +1,7 @@
+.. news-prs: 4134 4138 4151
+
+.. news-start-section: Additions
+- OpenDDS now stores configuration information in a key-value store.
+  - Configuration values can be set via API, config file, or command line.
+    - Currently applies to the `common` section and common transport configuration.
```

Comment:
```suggestion
- OpenDDS now stores configuration information in a key-value store.

  - Configuration values can be set via API, config file, or command line.
  
    - Currently applies to the ``common`` section and common transport configuration.
```

RST nested lists have to be separated by a blank line or else [they're not interpreted correctly](https://opendds--4151.org.readthedocs.build/en/4151/news.html#additions).

Also we could add a small section on this to the to [runtime config chapter](https://github.com/OpenDDS/OpenDDS/blob/master/docs/devguide/run_time_configuration.rst) in the devguide and link it here.

Reply:
That's true, I'm okay with that.